src/ApparentHorizons/Tags.hpp

```diff
@@ -215,6 +215,30 @@ struct Tangents : db::ComputeTag {
                                    Rhat<Frame>, Jacobian<Frame>>;
 };
 
+/// Computes the Euclidean area element on a Strahlkorper.
+/// Useful for flat space integrals.
+template <typename Frame>
+struct EuclideanAreaElement : db::ComputeTag {
+  static std::string name() noexcept { return "EuclideanAreaElement"; }
+  static constexpr auto function =
+      ::StrahlkorperGr::euclidean_area_element<Frame>;
+  using argument_tags = tmpl::list<
+      StrahlkorperTags::Jacobian<Frame>, StrahlkorperTags::NormalOneForm<Frame>,
+      StrahlkorperTags::Radius<Frame>, StrahlkorperTags::Rhat<Frame>>;
+};
+
+/// Computes the flat-space integral of a scalar over a Strahlkorper.
+template <typename IntegrandTag, typename Frame>
+struct EuclideanSurfaceIntegral : db::ComputeTag {
+  static std::string name() noexcept {
+    return "EuclideanSurfaceIntegral" + IntegrandTag::name();
```

Comment:
I think it's a rough convention to have the name for compute tags which are templated on other tags to have a form like `"EuclideanSurfaceIntegral(" + IntegrandTag::name() + ")"`